app/scripts/modules/core/src/application/ApplicationComponent.tsx

```diff
@@ -68,7 +68,7 @@ export class ApplicationComponent extends React.Component<IApplicationComponentP
           </div>
         )}
         <div className="container scrollable-columns">
-          <UIView className="secondary-panel" name="insight" />
+          <UIView className="secondary-panel" name="insight" app={app} />
```

Comment:
This isn't idiomatic use of UIView.  I'd prefer the `app` be wired using the resolve from the router because that's how we wire all the other resolve data right now